src/movies.js

```diff
@@ -1,16 +1,89 @@
 // Iteration 1: All directors? - Get the array of all directors.
+
+function getAllDirectors(movies){
+    let answer1 = movies.map((movie) => {
+        return movie.director;
+    })
+return answer1;
```

Comment:
Check indentation. This return should be at the same level as the last closing parentheses